src/write-commit-object

```diff
@@ -22,5 +22,5 @@ r.open(None)
 
 [_, rev] = r.resolve_rev(args.rev, True)
 [_, commit, _] = r.load_commit(rev)
-with open(args.builddir + '/ostree-commit', 'wb') as f:
+with open(args.builddir + '/ostree-commit-object', 'wb') as f:
```

Comment:
Will backcompat be a problem if we wholly rename it?  Or is the idea to change the usage of the commit object entirely in #515?

Reply:
Ahh OK, so this is not used elsewhere in cosa, but *is* used internally in the RHCOS pipeline to prepare for the next build. So yes, the code there would have to be slightly tweaked for the rename once RHCOS inherits these changes. Though I'd like to also upstream that S3 prep logic into the `buildprep` command to abstract all this.

Reply:
OK, thanks for confirming my thoughts.  I'm OK with this change regardless; we can limp along with `rhcos-4.1` for a while (at least until OpenShift 4 GA).  :wink: